ember_debug/adapters/basic.js

```diff
@@ -113,7 +113,7 @@ export default EmberObject.extend({
   */
   connect() {
     return new Promise((resolve, reject) => {
-      $(() => {
+      document.addEventListener('DOMContentLoaded', () => {
```

Comment:
The basic adapter code can actually run on all browsers, including IE, Safari, and mobile browsers.